.github/workflows/tag_trigger.yml

```diff
@@ -37,16 +37,8 @@ jobs:
       lambdaName: "All"
     secrets: inherit
 
-  run-twistlock:
-    needs: [setup-env, run-build]
-    uses: ./.github/workflows/twistlock.yml
-    with:
-      environment: "perf"
-      ref: "${{ needs.setup-env.outputs.git-tag }}"
-    secrets: inherit
-
   run-deployment:
-    needs: [setup-env, run-twistlock]
+    needs: [setup-env]
```

Comment:
This requires run-build as well I believe, since twistlock required it and this required twistlock.

Reply:
great catch!